src/fondant-chocolate-cake.md

```diff
@@ -0,0 +1,38 @@
+# Fondant Chocolate Cake
+
+A delicious chocolate cake that melts in your mouth. Very easy to make.
+
+- Prep time: 10 minutes.
+- Cook time: 30 minutes.
+- Servings: 6.
```

Comment:
```suggestion
- 🍽️ Servings: 6.
```
le emoji

Reply:
I am 100% against emojis in recipes, they are cringe, don't render properly in my editor because I didn't install the bloated fonts for them (and I won't because they are heavily politicized/cosmopolitan), and they are breaking a lot of my scripts, make conversion to latex nearly impossible, etc, etc. If this becomes a requirement I'm not going to bother opening PRs for my recipes anymore.